### PR TITLE
Cosmetic Drip 3.2 ~ Jumpsuits

### PR DESCRIPTION
**Pull request by KittenColony**
## About the PR
Adds more drip to Medical, including a resprite to the CMO turtleneck, a new utility suit and maint-surgeon outfit for MDs, a casual chemistry and psychologist outfit and a Medical Director suit for the CMO

## Why / Balance
More player expression!

## Technical details
2 changes to existing clothes, loadout code, new sprites in the _CD namespace

## Media
image

## Requirements

- [x] I have read and I am following the [Pull Request Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html). I understand that not doing so may get my pr closed at maintainer’s discretion
- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**
🆑 

add: Added a new casual outfits for Chemist and Psychologist
add: Added two new outfits for Medical Doctors
add: Added a formal suit for the CMO
resprite: Resprited the CMO turtleneck.

### CODE REVIEW

**Comment by KittenColony** [comment]
wouldn't be one of my PRs if it worked the first time.

File: Resources/Prototypes/Entities/Clothing/Uniforms/jumpskirts.yml [hunk 1]
```diff
@@ -126,9 +126,9 @@
   description: It's a turtleneck worn by those with the experience to be Chief Medical Officer. It provides minor biological protection.
   components:
   - type: Sprite
-    sprite: Clothing/Uniforms/Jumpskirt/cmo_turtle.rsi
+    sprite: _CD/Clothing/Uniforms/Jumpskirt/cmoturtleskirt.rsi # CD
   - type: Clothing
-    sprite: Clothing/Uniforms/Jumpskirt/cmo_turtle.rsi
+    sprite: _CD/Clothing/Uniforms/Jumpskirt/cmoturtleskirt.rsi # CD
```
Comment (dffdff2423) [inline Resources/Prototypes/Entities/Clothing/Uniforms/jumpskirts.yml:131]: Any reason why you are replacing these instead of creating a new item?

Comment (KittenColony) [reply Resources/Prototypes/Entities/Clothing/Uniforms/jumpskirts.yml:131]: its easier then replacing every reference to the original manually

**Review by Nairodian — APPROVED** [review]

File: Resources/Prototypes/Entities/Clothing/Uniforms/jumpskirts.yml (hunk 1 above)
Comment (Nairodian) [reply Resources/Prototypes/Entities/Clothing/Uniforms/jumpskirts.yml:131]: Yeah this is fine, it's extremely similar to the original design just done a little differently with different colors.

**Review by dffdff2423 — APPROVED** [review]

**Comment by dffdff2423** [comment]
Please fix merge conflicts

**Pull request merged**